.hlint.yaml

```diff
@@ -1 +1,4 @@
 - arguments: [--cpp-define=HLINT, --cpp-ansi]
```

Comment:
Just checking: is this `HLINT` directive actually needed? I ask since you've removed them elsewhere, which I support, but you left this one here for some reason.

Reply:
removed.